### PR TITLE
P0 screens

### PR DESCRIPTION
**Pull request by samyakshah3008**
FEAT: 

- Home Page
- Explore Page
- Bookmark Page
- User Profile Page
- Login Page
- Signup Page

image

image

image

image

image

image

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Top Accounts
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Hashtags
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
+            Locations
+          </Button>
+          <Button fontWeight="thin" variant="unstyled" fontSize="small">
```
Comment (hinanshisuthar) [inline src/components/Footer.jsx:42]: You have many buttons of same style. You can include them in array and map over them. Would help in reducing repetition. Just a suggestion

**Review by hinanshisuthar — COMMENTED** [review]
Amazing work done. Good you have used external library for styling. Clean and neat UI. Good to merge!

**Review by sharathnair9999 — APPROVED** [review]
Looks very clean. You can reduce number of commits per PR. Would consume less time for the reviewer. Just a suggestion 😅

**Review by lomesshh — APPROVED** [review]
clean and modular code
UI looks pretty cool
got some glimpses of chakra UI from your code

looks good to merge

File: src/components/Footer.jsx (hunk 1 above)
Comment (samyakshah3008) [reply src/components/Footer.jsx:42]: Good point, will update it soon
Thanks!

**Pull request merged**